react/ProductDetails.js

```diff
@@ -89,140 +89,178 @@ class ProductDetails extends Component {
       : (otherQuantity === 0 && -1) || price - otherPrice
   }
 
+  renderLoader() {
+    const { displayVertically } = this.props
+
+    const isVertical = displayVertically ? 'VERTICAL' : 'HORIZONTAL'
```

Comment:
Isn't better to call this entry `orientation`? `isVertical` looks like a boolean content.